### DISCUSSION TITLE
How to update latest PR locally for Openroad flow scripts

### DISCUSSION THREAD

**Discussion by vijayank88** [The-OpenROAD-Project/OpenROAD-flow-scripts#153] [Q&A] [answered]
I've installed ORFS with --local already.

Is there any specific steps there to update/merge latest PR ?

I have followed below commands and get struck with Error -132
#git checkout master
#git pull
#./build_openroad.sh --local

Is that right or only git pull enough to merge latest PR updates?

**Comment by maliberty** [comment, marked as answer]
you can use --latest to get the head of the master branch of OR

**Reply by vijayank88** [reply]
@maliberty can you please give those steps in detail. Thanks

**Comment by vvbandeira** [comment]
```
# go to master branch
git checkout master

# get changes from upstream
git fetch --all

# update local commits with upstream
git merge origin/master

# remove any previous build artifacts
git clean -xdf ./tools

# build again

## build using the latest OpenROAD commit
./build_openroad.sh --local --latest

## build with latest stable OpenROAD commit
git submodule update
./build_openroad.sh --local
```
